src/views/Picklists.vue

```diff
@@ -12,6 +12,16 @@
     </ion-header>
     
     <ion-content :fullscreen="true">
+      <ion-list v-if="completedPicklist.length">
+        <ion-list-header lines="none">
+          <ion-label>{{ $t("Completed") }}</ion-label>
```

Comment:
Internationalize the text.

Reply:
Sir, Completed was already there in the en.json file.
Thank you Sir for reviewing my code.

Reply:
Okay